### PR TITLE
LPS-199483 Folders with long names cannot be selected in Model Builder

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPS-199483](https://liferay.atlassian.net/browse/LPS-199483)

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @cgmonte ,
I'm assigning this PR to you because it is frontend. Feel comfortable assigning to anybody from your squad :+1:

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss [hunk 1]
```diff
@@ -25,6 +25,10 @@
 		gap: 1rem;
 		width: 100%;
 
+		&-title {
+			min-width: 3.125rem;
```
Comment (cgmonte) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss:29]: Hi @larissacribeiro, this change sure fixes the bug (now we can select the folder), but it creates another, visual one. See bellow. I think that adding the `text-truncate` class to the `span` element is a better fix. Have you tried that?
screenshot-1698098961

**Review by cgmonte — COMMENTED** [review]
Hi @larissacribeiro, thanks for working on this! I have made a comment. Please let me know what you think.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss (hunk 1 above)
Comment (larissacribeiro) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss]: You're right, just did it!

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:sf

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by cgmonte** [comment]
ci:forward

**Comment by larissacribeiro** [comment]
ci:test:stable

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by victorhcunha** [comment]
ci:reevaluate:1307401_8185

**Comment by larissacribeiro** [comment]
Manually forwarded toBrian here: https://github.com/brianchandotcom/liferay-portal/pull/142836

**Pull request closed without merging**